### PR TITLE
docs: update readme to follow 1.9 updates

### PR DESCRIPTION
**Pull request by converge**
Signed-off-by: João Vanzuita <joao@kubeshop.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +88,24 @@ kubefirst init \
...
+kubefirst init \
+--admin-email email@example.com.com \
+--cloud aws \
+--hosted-zone-name example.com \
+--region eu-central-1 \
+--cluster-name example_com \
+--profile default \
+--github-user example-github-username \
+--github-org example-organization \
+--gitops-branch main \
```
Comment (6za) [inline README.md:105]: this choices are not expected for users to use, unless a bug is detected..

Comment (6za) [reply README.md:105]: `--gitops-branch` and `--metaphor-branch main`

Comment (converge) [reply README.md:105]: few days ago I tried without these flags and installation failed. we could merge it like this, and update asap the new tags are created to fix this.

Comment (6za) [reply README.md:105]: it will only work, once we tag 1.9.0, until there we need to pass it.

**Review by johndietz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (johndietz) [reply README.md:105]: hey fellas, we can course correct in the morning, these docs won't publish to prod until we're ready, but getting prepped for prod and it looks like i want these in. forgive me and help me adjust if i'm overstepping with the merge and we can adjust before prod publish. @converge @6za

**Pull request merged**